### PR TITLE
Fixed register UI, added logo image in login and register

### PR DESCRIPTION
**Pull request by adrean-hipe**
Also added some * on required fields of login and register, created comments in login and register sections

### CODE REVIEW

File: laravel-app/routes/web.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return redirect()->route('login');
 });
```
Comment (hipe01mark) [inline laravel-app/routes/web.php:18]: Create LoginController to it. Do not expose the backend process in web routes!

ref:
https://laravel.com/docs/10.x/controllers#main-content

File: laravel-app/resources/views/auth/login.blade.php [hunk 2]
```diff
@@ -4,14 +4,21 @@
...
+                <h1 class="ml-0">icroblog</h1>
+            </div>
+
+            <!-- Login card -->
             <div class="card m-3">
                 <div class="card-body">
                     <form method="POST" action="{{ route('login') }}">
                         @csrf
+
+                        <!-- Email input field -->
```
Comment (bpocrafael) [inline laravel-app/resources/views/auth/login.blade.php:18]: remove comments

File: laravel-app/resources/views/auth/login.blade.php [hunk 3]
```diff
@@ -55,17 +65,23 @@
                             </div>
                         </div>
 
+                        <!-- Login submit button -->
```
Comment (bpocrafael) [inline laravel-app/resources/views/auth/login.blade.php:68]: remove unnecessary comments

File: laravel-app/routes/web.php [hunk 4]
```diff
@@ -19,4 +20,4 @@
 
 Auth::routes();
 
-Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home');
+Route::get('/', [HomeController::class, 'index'])->name('home');
```
Comment (hipe01mark) [inline laravel-app/routes/web.php:23]: Add new line at the end of the code

**Review by hipe01mark — APPROVED** [review]
LGTM but I have 1 minor comment 🙇

File: laravel-app/routes/web.php (hunk 4 above)
Comment (bpocrafael) [inline laravel-app/routes/web.php:23]: add newline

**Review by bpocrafael — APPROVED** [review]
lgtm :shipit:

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

File: laravel-app/resources/views/auth/login.blade.php [hunk 5]
```diff
@@ -4,14 +4,20 @@
 <div class="container">
     <div class="row justify-content-center">
         <div class="col-md-4">
-            <h1 class="text-center p-5">Microblog</h1>
+            <div class="d-flex justify-content-center align-items-center text-center p-4">
+                <img class="img-fluid w-25" src="{{ asset('microblog-logo.png') }}" alt="Microblog Logo">
+                <h1 class="ml-0">icroblog</h1>
+            </div>
+
+            <!-- Login card -->
```
Comment (bpocrafael) [inline laravel-app/resources/views/auth/login.blade.php:12]: I was wondering maybe you could create a component for a card.

https://laravel.com/docs/8.x/blade#rendering-components

**Pull request merged**